src/components/StockDetails/StockPredictionOverview.jsx

```diff
@@ -1,64 +1,16 @@
 import React from 'react'
-import {Box, Grid, GridItem, Heading, SimpleGrid, Spacer, useColorModeValue, VStack} from "@chakra-ui/react";
+import {Box, useColorModeValue} from "@chakra-ui/react";
 import {Line} from "react-chartjs-2";
 import {useStockSymbol} from "../../contexts/useStockInfo";
 
-const CustomGridItem = ({children, ...otherProps}) => (
-    <GridItem mx={4} my={4} borderRadius="lg" shadow="md" {...otherProps}>
-        {children}
-    </GridItem>
-)
-
-const CustomContainer = ({children, ...otherProps}) => (
-    <Box p={4} w="80%" textAlign="center" bgColor={useColorModeValue("brand.300", "brand.600")}
-         borderRadius="lg" {...otherProps}>
-        {children}
-    </Box>
-)
-
-const PredictionTable = () => (
-    <CustomGridItem colSpan={2} shadow="none">
-        <VStack spacing={4}>
-            <CustomContainer>
-                Prediction
-            </CustomContainer>
-            <CustomContainer p={4} w="80%" bgColor={useColorModeValue("brand.300", "brand.600")} borderRadius="lg">
-                <VStack spacing={4}>
-                    <Box p={4} w="100%" bgColor={useColorModeValue("brand.100", "brand.500")} borderRadius="lg">
-                        <SimpleGrid columns={{base: 1, lg: 2}} spacing={1}>
-                            <Heading as="h3" size="lg">1 Day</Heading>
-                            <Heading as="h3" size="lg" color="lightgreen">612.25</Heading>
-                        </SimpleGrid>
-                    </Box>
-                    <Box p={4} w="100%" bgColor={useColorModeValue("brand.100", "brand.500")} borderRadius="lg">
-                        <SimpleGrid columns={{base: 1, lg: 2}} spacing={1}>
-                            <Heading as="h3" size="lg">1 Week</Heading>
-                            <Heading as="h3" size="lg" color="lightgreen">676.25</Heading>
-                        </SimpleGrid>
-                    </Box>
-                </VStack>
-            </CustomContainer>
-        </VStack>
-    </CustomGridItem>
-)
-
-const PredictionGraph = () => {
+const StockDataGraph = () => {
     const {graphData} = useStockSymbol()
     return (
-        <CustomGridItem colSpan={3}>
+        <Box w='80%' mt={8} p={4} bgColor={useColorModeValue("brand.100", "brand.200")} borderRadius='xl'
```

Comment:
dark mode background is too bright. would rather have a constant black/ dark grey background (and white text)

Reply:
of course best thing is to have bgColors that suit the theme

Reply:
I fixed it and pushed to this branch